.github/workflows/ci-tests.yml

```diff
@@ -141,6 +146,7 @@ jobs:
       - name: Run CWL conformance tests ${{ matrix.cwl-version }}
         env:
           version: ${{ matrix.cwl-version }}
+          spec_branch: exclude
```

Comment:
This branch was merged into `main`
```suggestion
```

Reply:
Thanks - I'll do PRs for the other spec repos then can replace this.

Reply:
done